src/networkAddresses.ts

```diff
@@ -36,5 +36,8 @@ export const NETWORK_ADDRESSES = {
     VUSDT: "0x2c3Cc8EED38b4d323ce97DB11f98737F3F7421A1",
     VUSDC: "0x6f1089e90C6a14397C20348eE704d596819DFEd4",
     VTREASURY: "0x4116CA92960dF77756aAAc3aFd91361dB657fbF8",
+    XVS_VAULT_PROXY: "0x486E12d255BE7337309916CCE17302CC38D11599",
+    XVS_STORE: "0xE5dec7f74A370E77751fC64F084f4074e7F2CcEc",
```

Comment:
There are new addresses for these contracts in https://github.com/VenusProtocol/venus-protocol/pull/366

Reply:
[a01da56](https://github.com/VenusProtocol/vips/pull/112/commits/a01da56458a4d2340f89f1478a031eaa6338c5dc)